### ISSUE TITLE
Update like FireFox about section

### ISSUE THREAD

**Issue by Chandlr** [x64dbg/x64dbg#505]
It would be sexy as fuck to have a update function like FireFox :)
The i guess people would update more often aswell

**Comment by mrexodia** [comment]
They would (just like with Firefox :smile:) be spammed with updates almost every day but a direct link the latest release archive in the update checker would work for me. In addition a weekly/whatever update check would help people with getting new releases.

**Comment by G3ph4z** [comment]
Damn, this 9 years old. 😅 Luckily, the ("Check for updates") feature is implemented and works like a charm, so I think it can be closed, It'll be a historical moment tho.

**Comment by mrexodia** [comment]
Yeah, but it doesn’t update automatically which was the point of this issue…

**Comment by G3ph4z** [comment]
> Yeah, but it doesn’t update automatically which was the point of this issue…

That's right, but the solution you have suggested is already implemented:

> They would (just like with Firefox 😄) be spammed with updates almost every day but a direct link the latest release archive in the update checker would work for me. In addition a weekly/whatever update check would help people with getting new releases.
